src/request/transaction.ts

```diff
@@ -117,12 +117,16 @@ export default class Transaction {
   private sanitizeKeys() {
     const sanitized = Object.assign({}, this) as any;
 
-    if (
-      sanitized.creditCard != null &&
-      sanitized.creditCard.last4digits != null
-    ) {
-      sanitized.creditCard.last_4_digits = this.creditCard!.last4digits;
-      delete sanitized.creditCard.last4digits;
+    if (sanitized.creditCard != null) {
+      if (sanitized.creditCard.last4digits != null) {
+        sanitized.creditCard.last_digits = this.creditCard!.last4digits;
+        delete sanitized.creditCard.last4digits;
+      }
+
+      if (sanitized.creditCard.lastDigits != null) {
```

Comment:
I don't _think_ this is needed. I believe that it was needed for `last4digits` as `snakecaseKeys` didn't handle that case correctly.